plugins/pa11y/src/tasks/pa11y.ts

```diff
@@ -10,7 +10,13 @@ export default class Pa11y extends Task<typeof Pa11ySchema> {
   static description = ''
 
   async run(): Promise<void> {
+    const localState = readState('local')
     const reviewState = readState('review')
+
+    if (localState) {
+      process.env.TEST_URL = `https://local.ft.com:${localState.port}`
+    }
+
     if (reviewState) {
```

Comment:
**nitpick**: using an `else if` conditional would make it clearer that the `TEST_URL` will be overwritten if `reviewState` is truthy, even if `localState` is also truthy
```suggestion
    } else if (reviewState) {
```